### PR TITLE
fix bug 1310510 - make install instructions more user-friendly

### PR DESCRIPTION
**Pull request by bychek-ru**
@mathjazz @jotes r?

### CODE REVIEW

File: docs/dev/install.rst [hunk 1]
```diff
@@ -86,15 +86,28 @@ Installation
 
    - ``SITE_URL`` should be set to the URL you will use to connect to your
      local development site. Some people prefer to use
-     ``http://127.0.0.1:8000`` instead of ``localhost``.
+     ``http://127.0.0.1:8000`` instead of ``localhost``. However, should you
+     decide to change the ``SITE_URL``, you also need to
+     [request](https://developer.mozilla.org/en-US/docs/Mozilla/Tech/Firefox_Accounts/Introduction)
```
Comment (mathjazz) [inline docs/dev/install.rst]: Maybe move the link out of the paragraph for easier reading and consistency with the rest of the document? Check out the links section under "The site should be available at http://localhost:8000."

**Comment by mathjazz** [comment]
Good job! r+

File: docs/dev/install.rst (hunk 1 above)
Comment (jotes) [inline docs/dev/install.rst]: I think we should mention that our demo/intro site (http://localhost:8000/intro) will require change of base url if someone will decide to change SITE_URL.

**Comment by bychek-ru** [comment]
@mathjazz Updated

**Comment by mathjazz** [comment]
Thanks!

**Pull request merged**